plugins.xml

```diff
@@ -3761,6 +3761,31 @@
 			<certification type="official" />
 			<description>Compatibility release of the JATS Template plugin for OJS 3.3.0</description>
 		</release>
+		<release date="2024-03-11" version="1.0.6.2" md5="ec611cf64a6e74cc04a336c2105576ab">
+			<package>https://github.com/pkp/jatsTemplate/releases/download/v1_0_6-2/jatsTemplate-v1_0_6-2.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>3.3.0.1</version>
+				<version>3.3.0.2</version>
+				<version>3.3.0.3</version>
+				<version>3.3.0.4</version>
+				<version>3.3.0.5</version>
+				<version>3.3.0.6</version>
+				<version>3.3.0.7</version>
+				<version>3.3.0.8</version>
+				<version>3.3.0.9</version>
+				<version>3.3.0.10</version>
+				<version>3.3.0.11</version>
+				<version>3.3.0.12</version>
+				<version>3.3.0.13</version>
+				<version>3.3.0.14</version>
+				<version>3.3.0.15</version>
+				<version>3.3.0.16</version>
+				<version>3.3.0.17</version>
+				<version>~3.3.0.0</version>
```

Comment:
The semver-style compatibility statements were supported starting with 3.3.0-11. I'd suggest removing the 3.3.0-11 to 3.3.0-17 statements in favour of this one.

Reply:
No problem. I was wondering what the preferred approach for that was. Thanks!